runtime/runtime/src/actions.rs

```diff
@@ -1015,6 +1061,31 @@ pub(crate) fn check_account_existence(
     Ok(())
 }
 
+fn check_transfer_to_nonexisting_account(
+    config: &RuntimeConfig,
+    is_the_only_action: bool,
```

Comment:
I think we need to revisit the `is_the_only_action` logic because now we have two kinds of transfers. Maybe this should be changed to `only_transfers`? Or Maybe there is no use-case for creating an implicit account with both normal and non-refundable balances in the one transaction, but then we will need to document this clearly.